code/modules/cm_marines/Donator_Items.dm

```diff
@@ -1366,6 +1362,12 @@
 	icon_state = "securitypack"
 	item_state = "securitypack"
 
+/obj/item/storage/backpack/marine/satchel/fluff/sas_juggernaut //CKEY=sasoperative (UNIQUE)
+	name = "Tactical Radiopack"
```

Comment:
Names are lower case unless it's a proper noun, which this isn't

Reply:
Understood, changes made!